### ISSUE TITLE
Anti-DNA autoantibodies positivity

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4861]
**Preferred term label:**
Anti-DNA autoantibodies positivity

**Synonyms**
Anti-DNA double strand positivity

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against DNA
PMID: 15804704 

**Parent term (use hpo.jax.org/app)**
http://purl.obolibrary.org/obo/HP_0003493

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM 152700
SYSTEMIC LUPUS ERYTHEMATOSUS; SLE

**Your nano-attribution (ORCID)**

0000-0001-9969-8610
